core/src/components/searchbar/usage/angular.md

```diff
@@ -43,3 +43,13 @@
   <ion-searchbar></ion-searchbar>
 </ion-toolbar>
 ```
+
+### Showing the proper keyboard submit button
+
+#### Android
+
+By default, tapping `ion-searchbar` will cause the keyboard to appear with a magnifying glass icon on the submit button. You can optionally set `inputmode="search"`, which will change the icon from a magnifying glass to a carriage return.
+
+#### iOS
+
+By default, tapping `ion-searchbar` will cause the keyboard to appear with the text "return" on a gray submit button. You can optionally set `inputmode="search"`, which will change the text from "return" to "go", and change the button color from gray to blue. Additionally, you can wrap the `ion-searchbar` in a `form` element with an `action` property. This will cause the keyboard to appear with a blue submit button that says "search".
```

Comment:
```suggestion
By default, tapping `ion-searchbar` will cause the keyboard to appear with the text "return" on a gray submit button. You can optionally set the `inputmode` property to `"search"`, which will change the text from "return" to "go", and change the button color from gray to blue. Alternatively, you can wrap the `ion-searchbar` in a `form` element with an `action` property. This will cause the keyboard to appear with a blue submit button that says "search".
```